pkg/processor/trigger/partitioned/v3io/partition.go

```diff
@@ -131,3 +136,29 @@ func (p *partition) getRecords(partitionPath string, location string, pollingInt
 	return response.Output.(*v3iohttp.GetRecordsOutput), nil
 
 }
+
+func (p *partition) waitPartitionAvailable(partitionPath string, pollingInterval time.Duration) error {
+	responseChannel := make(chan *v3iohttp.Response)
+
+	for {
+		request := &v3iohttp.ListBucketInput{
+			Path: p.v3ioTrigger.streamPath,
+		}
+		p.v3ioTrigger.container.ListBucket(request, context.TODO(), responseChannel)
+		response := <-responseChannel
+
+		listBucketResult, ok := response.Output.(*v3iohttp.ListBucketOutput)
+		if !ok {
```

Comment:
I'm not sure this check it comprehensive. Shouldn't we check err anyway? I think `response.Error` being set and `response.Output` nil, `ok` would be true and skip the error case.